ush/config.sh.RRFS_dev1

```diff
@@ -21,6 +21,7 @@ FIX_GSI=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_gsi
 FIX_CRTM=/home/rtrr/FIX_EXEC_MODULE/crtm/CRTM_v2.3.0
 OBSPATH_NSSLMOSIAC=/public/data/radar/nssl/mrms/conus
 RADARREFL_TIMELEVEL=(0 15 30 45)
+#FH_DFI_RADAR=(0.0,0.25,0.5)
```

Comment:
Do you mean `FH_DFI_RADAR=(0.0 0.25 0.5)` instead of `FH_DFI_RADAR=(0.0,0.25,0.5)` (the latter is a one element array, i.e. `"0.0,0.25,0.5"` ) ?

Reply:
It has to include comma as it will be used to fill in namelist variable.

Reply:
Okay, I see. In this case, to avoid confusion, would it be enough to just let `FH_DFI_RADAR="0.0,0.25,0.5"` and hence no need to deal with an array which contains only one item?

Reply:
That doesn't not work when python generates the namelist for FV3.